src/google/protobuf/compiler/csharp/csharp_wrapper_field.cc

```diff
@@ -196,6 +219,15 @@ void WrapperOneofFieldGenerator::GenerateSerializationCode(io::Printer* printer)
     "}\n");
 }
 
+void WrapperOneofFieldGenerator::GenerateAsyncSerializationCode(io::Printer* printer) {
+  // TODO: I suspect this is wrong...
```

Comment:
It's fine, I believe. I should remove the comment elsewhere, but let's not include this one :)